pkg/network/node/ovscontroller.go

```diff
@@ -199,16 +199,21 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	// eg, "table=90, priority=100, ip, nw_dst=${remote_subnet_cidr}, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31], set_field:${remote_node_ip}->tun_dst,output:1"
 	otx.AddFlow("table=90, priority=0, actions=drop")
 
-	// Table 100: egress routing; edited by SetNamespaceEgress*()
-	otx.AddFlow("table=100, priority=300,udp,udp_dst=%d,actions=drop", vxlanPort)
-	otx.AddFlow("table=100, priority=200,tcp,tcp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
-	otx.AddFlow("table=100, priority=200,udp,udp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
-	otx.AddFlow("table=100, priority=150,ct_state=+rpl,actions=goto_table:101")
-	// eg, "table=100, priority=100, reg0=${tenant_id}, ip, actions=set_field:${tun0_mac}->eth_dst,set_field:${egress_mark}->pkt_mark,goto_table:101"
+	// Table 99: legacy DNS rules needed to override egress IP and egress
+	// network policy, from 3.11. These are used in case someone has configured
+	// pod DNS to talk to the node IP, and also has an EgressNetworkPolicy
+	// saying "Deny 0.0.0.0/0", then DNS to the node IP is still expected to
+	// work
+	otx.AddFlow("table=99, priority=300,udp,udp_dst=%d,actions=drop", vxlanPort)
+	otx.AddFlow("table=99, priority=200,tcp,tcp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
+	otx.AddFlow("table=99, priority=200,udp,udp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
```

Comment:
You need to change the `actions=goto_table:100` rule in table 30 to `goto_table:99` now. Also, you need

    otx.AddFlow("table=99, priority=0, actions=goto_table:100")

at the end. Or, alternatively, keep these rules as table 100 (and do `goto_table:101` at the end), have ENP at 101, and have egress routing at 102. That would be slightly more consistent with how we number things elsewhere but I don't have a strong preference.

Reply:
If you're touching these lines can you add spaces after all the commas in the flow to make them consistent with the rest of the file.

Also, the comment about DNS applies to the two `priority=200` rules, but the `vxlanPort` rule is an unrelated security fix. Actually, you could move that to the top of table 20 if you wanted, since it only applies to traffic coming out of pods.

Reply:
OK, I've done:

- moved the `vxlanPort` rule to the top of table 20
- changed the `actions=goto_table:100` rule in table 30 to `goto_table:99` (and hence kept table 99) 
- Added the missing flow: `otx.AddFlow("table=99, priority=0, actions=goto_table:100")`
- Added spaces after all the commas 

Thanks for spotting these!